src/js/client.js

```diff
@@ -101,7 +101,7 @@ class Client {
       return false
     }
 
-    console.log("making move: " + JSON.stringify(move))
+   // console.log("making move: " + JSON.stringify(move))
```

Comment:
add back comment? was the problems all the ticks? those don't log clientside in my branch